### ISSUE TITLE
Forms: List with grouped fields does save values, but does not fetch values

### ISSUE THREAD

**Issue by pamtbaau** [getgrav/grav#1998]
I have created a Test plugin and use an Admin form for plugin settings as well as a form for settings per Page.

I noticed that if I use a List field which embeds grouped fields like: `.group.field`, the values are saved properly, but don't get fetched.

**user/plugins/test/blueprints.yaml:**
```
form:
  validation: strict
  fields:
    group1.field1:
      type: text
      label: Label group1.field1
      placeholder: Placeholder for group1.field1

    group1.field2:
      type: text
      label: Label group1.field2
      placeholder: Placeholder for group1.field2

    listname:
      type: list
      label: Label for list
      fields:
        .field3:
          type: text
          label: Label field3
          placeholder: Placeholder for .field3 (not group)
        .group2.field4:
          type: text
          label: Label group2.field4
          placeholder: Placeholder for .group2.field4
        .group2.field5:
          type: text
          label: Label group2.field5
          placeholder: Placeholder for .group2.field5
```

When entering data at `/admin/plugins/test` the form looks as follows:
![listform-save](https://user-images.githubusercontent.com/5120011/39355175-ba446984-4a0c-11e8-8885-97337a38d3bd.png)

After saving, the grouped fields inside the list are not being fetched:
![listform-fetch](https://user-images.githubusercontent.com/5120011/39355279-0f46d6a6-4a0d-11e8-90d8-58bf39bf8e72.png)

The generated Yaml is however as expected.
**user/config/plugins/test.yaml**
```
group1:
  field1: a
  field2: b
listname:
  -
    field3: c
    group2:
      field4: d
      field5: e
```

Same happens when editing a page in Admin.

Is this an issue in Grav or an oversight by me?

**Comment by rhukster** [comment]
I don't think you can set list items with nested fields.  The results bear this out:

```
ields:
        .field3:
          ..works..
        .group2.field4:
          --no work---
        .group2.field5:
          --no work--
```

Basically everything at the list level needs to be at the 'same' level.  You might need to restructure your YAML to reflect this.

**Comment by pamtbaau** [comment]
The result shows that saving the nested structure works as expected. It's only the reading that fails...

Oh well... Being a bit ugly, but renaming the groups into group2_field4 will do the job for now :-)

**Comment by mahagr** [comment]
This has been likely fixed in Admin 1.10 as I just fixed related bug and everything seems to work.
